src/app/app.module.ts

```diff
@@ -35,6 +33,10 @@ import { FaqCategoryFormComponent } from './faqs/faq-category-form/faq-category-
 import { FaqFormDialogComponent } from './faqs/faq-form-dialog/faq-form-dialog.component';
 import { FaqFormComponent } from './faqs/faq-form/faq-form.component';
 import { FaqAccordionComponent } from './faqs/faq-accordion/faq-accordion.component';
+import { KToolCardComponent } from './k-tools/k-tool-card/k-tool-card.component';
```

Comment:
is this a display/view of a single item? We should use something like k-tool-display, k-tool-item or k-tool. Having "card" in there implies a html styling that could change throughout the life of the app.

Reply:
I agree. I like k-tool-item